index.html

```diff
@@ -1338,6 +1338,19 @@ <h2>
 key formats for RSA MUST cause the DID Document Processor to raise an error.
             </td>
           </tr>
+          <tr>
+            <td>
+secp256k1
+            </td>
+            <td>
+Secp256k1 key values MAY be expressed as the raw 32-byte value encoded in
+base58-btc format using the <code>publicKeyBase58</code> property or
+MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
+property. Both the raw 32-byte base58-btc format and JWK format MUST be
```

Comment:
s/base58-btc/base58/

Reply:
There are two base58 encodings... Bitcoin's and Flickr's... We need to be specific about which one we're using (most use Bitcoin's base58 encoding scheme).

Reply:
There is now a specific ref here: https://tools.ietf.org/html/draft-msporny-base58